sdcm/nemesis.py

```diff
@@ -32,21 +32,23 @@
 from sdcm.cluster_aws import ScyllaAWSCluster
 from sdcm.cluster import SCYLLA_YAML_PATH, NodeSetupTimeout, NodeSetupFailed, Setup
 from sdcm.mgmt import TaskStatus
-from .utils import get_data_dir_path, retrying
+
+from .utils import get_data_dir_path, retrying, remote_get_file
 from .log import SDCMAdapter
 from .keystore import KeyStore
 from . import prometheus
 from . import mgmt
 from avocado.utils import wait
 from . import wait as wait_wrap
 
-
-from sdcm.utils import remote_get_file
 from sdcm.sct_events import DisruptionEvent, DbEventsFilter
 
 
 class Nemesis(object):
 
+    disruptive = False
```

Comment:
why two params?

Reply:
We have old nemesis and deprecated. Adding 2 parameters allow explicitly define group which nemesis is belonged.

Reply:
per our discussion: remove this and create a list of "deprecated" nemesis

Reply:
As we discussed, remove nondisruptive flag.These cause to add 2 additional lists, which contain Complex Nemesis such as Limited, Chaos etc and Deprecated.